src/gateway/kong-production/monitoring/prometheus.md

```diff
@@ -1,6 +1,130 @@
+* [Docker](https://docs.docker.com/get-docker/) is used to run {{site.base_gateway}}, the supporting database, 
+and Prometheus locally. 
+* [curl](https://curl.se/) is used to send requests to {{site.base_gateway}}. `curl` is pre-installed on most systems.
+
+### Configure Prometheus monitoring
+
+1. Run the following command to start {{site.base_gateway}} using Docker:
+
+   ```sh
+   curl -Ls get.konghq.com/quickstart | sh -s -- -m
+   ```
+
+   You should see the following message when {{site.base_gateway}} is ready:
+
+   ```text
+   ✔ Kong is ready!
+   ```
+
+   The `-m` flag instructs the script to install a mock service to make testing easier. 
+   You will use this service later to generate sample metrics. You can send a sample request with:
+
+   ```sh
+   curl -s localhost:8000/mock/requests
+   ```
+
+1. Install the Prometheus {{site.base_gateway}} plugin:
+
+   ```sh
+   curl -s -X POST http://localhost:8001/plugins/ \
+    --data "name=prometheus" 
+   ```
+
+   You should receive a JSON response with the details of the installed plugin.
+
+1. Create the following Prometheus configuration file named `prometheus.yml`
+in the current directory:
+
+   ```text
+   scrape_configs:
+     - job_name: 'kong'
+       scrape_interval: 5s
+       static_configs:
+         - targets: ['kong-quickstart-gateway:8001']
+   ```
+
+   See the Prometheus [Configuration Documentation](https://prometheus.io/docs/prometheus/latest/configuration/configuration/)
+   for details on these settings.
+
+1. Run a Prometheus server passing it the configuration file created previously. Prometheus 
+will begin to scrape metrics data from {{site.base_gateway}}.
+
+   ```sh
+   docker run -d --name kong-quickstart-prometheus \
+      --network=kong-quickstart-net -p 9090:9090 \
+      -v $(PWD)/prometheus.yml:/etc/prometheus/prometheus.yml \
+      prom/prometheus:latest
+   ```
+
+1. Generate sample traffic to the mock service. This allows you to observe 
+   metrics generated from the StatsD plugin. The following command generates 60 
+   requests over one minute. Run the following in a new terminal:
+
+   ```bash
+   for _ in {1..60}; do {curl -s localhost:8000/mock/request; sleep 1; } done
+   ```
+
+1. You can view the metric data directly from {{site.base_gateway}} by querying the
+   `/metrics` endpoint on the [Admin API](/gateway/{{page.gateway_version}}/admin-api/):
+
+   ```sh
+   curl -s localhost:8001/metrics
+   ```
+
+   {{site.base_gateway}} will report system wide performance metrics by default. 
+   When the Plugin has been installed and traffic is being proxied, it will record 
+   additional metrics across service, route, and upstream dimensions.
+
+   See the [Kong Prometheus Plugin documentation](https://prometheus.io/docs/prometheus/latest/configuration/configuration://docs.konghq.com/hub/kong-inc/prometheus/)
+   for details on the available metrics and configurations.
+
+1. Prometheus provides multiple ways to query collected metric data. 
```

Comment:
Would this be worth splitting into another H2?

Reply:
IMO I don't see the need but don't feel strongly.  The statsD page is one section of instructions. If you feel strongly I'll add it, lmk

Reply:
ok, lets leave it alone